src/Datastore/RAMDataStore.h

```diff
@@ -0,0 +1,47 @@
+/** ----------------------------------------------------------------------------
+ * @file    RAMDataStore.h
+ * @author  Samuel Hick <samuel.hick@gmx.at>
+ * @brief   Class RAMDataStore definition
+ * ---------------------------------------------------------------------------*/
+
+#ifndef _RAMDATASTORE_H_
+#define _RAMDATASTORE_H_
+
+#include <mutex>
+#include <shared_mutex>
+#include <string>
+#include <variant>
+#include <vector>
+
+#include "Datastore/DataStore.h"
+#include "Types/GlobalTypes.h"
+#include "Types/Queue.h"
+#include "Types/Result.h"
+#include "Types/Tracks.h"
+#include "User.h"
```

Comment:
move the `User` files to the `Types/` folder

Reply:
done